python/ray/experimental/sgd/modified_allreduce.py

```diff
@@ -0,0 +1,627 @@
+# Copyright 2017 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Can you add a comment at the top of all the copied files saying something like "This file was copied from [RELEVANT URL]"

Reply:
Done